canary/webrtc-c/src/Peer.cpp

```diff
@@ -379,6 +383,17 @@ STATUS Peer::connect()
     STATUS retStatus = STATUS_SUCCESS;
     CHK_STATUS(signalingClientConnectSync(signalingClientHandle));
 
+    if(this->useMediaStorage){
+        // Join storage session for media ingestion
+        std::cout << "[KVS Storage Master]invoking join storage session" << endl;
```

Comment:
Just want to make sure the logging is consistent - I see in other places in this file that DLOGW is used